mcrouter/scripts/docker/container/ubuntu-packaged/usr/local/bin/entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -x
+
+
+function write_default_config() {
+cat << EOF > /etc/mcrouter.json
+{
+  "pools": {
+    "A": {
+      "servers": [
+        "127.0.0.1:5001"
+      ]
+    }
+  },
+  "route": "PoolRoute|A"
+}
+EOF
+}
+
+function run_mcrouter() {
+        local PORT=$1
+        mcrouter \
+        --config file:/etc/mcrouter.json
+#        -p "${PORT}"
+}
+
+
+if [ -z "${MCROUTER_LISTEN_PORT}" ]; then
+  echo "MCROUTER_LISTEN_PORT unset! Defaulting to port 5000 ..."
+  export LISTEN_PORT=5000
+  if [ ! -f "/etc/mcrouter.json" ]; then
```

Comment:
I would move this nested if statement out of the port check. If MCROUTER_LISTEN_PORT is set and /etc/mcrouter.json is not set it will error since write_default_config will not get called.